expo-maps/ios/ExpoMaps/GoogleMaps/PlacesUtils/GoogleMapsPlacesSearchCompleter.swift

```diff
@@ -0,0 +1,76 @@
+import GooglePlaces
+import GoogleMaps
+import ExpoModulesCore
+
+class GoogleMapsPlacesSearchCompleter: SearchCompleter {
+  
+  private var placesClient: GMSPlacesClient
+  private var tokenUtils: GoogleMapsPlacesTokenUtils
+  private var searchCompleterResults: [GMSAutocompletePrediction]?
+  private var filter = GMSAutocompleteFilter()
+  
+  init(placesClient: GMSPlacesClient, tokenUtils: GoogleMapsPlacesTokenUtils) {
+    self.placesClient = placesClient
+    self.tokenUtils = tokenUtils
+  }
+  
+  func autoComplete(searchQueryFragment: String) {
+    autoComplete(searchQueryFragment: searchQueryFragment, promise: nil)
+  }
+  
+  func autoComplete(searchQueryFragment: String, promise: Promise?) {
+    guard let token = tokenUtils.getToken() else {
+      print("No token provided for auto complete request!")
```

Comment:
I would recommend passing this string to the exception and not printing it here.